sys/freebsd/dev_crypto.txt

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <sys/time.h>
+include <fcntl.h>
+include <opencrypto/cryptodev.h>
+
+resource fd_crypto[fd]
+
+openat$crypto(fd const[AT_FDCWD], file ptr[in, string["/dev/crypto"]], flags flags[open_flags], mode const[0]) fd_crypto
+
+define CRIOGET	100
+ioctl$CRIOGET(fd fd_crypto, cmd const[CRIOGET], arg ptr[in, int32])
+ioctl$CIOCFINDDEV(fd fd_crypto, cmd const[CIOCFINDDEV], arg ptr[in, crypt_find_op])
+ioctl$CIOCGSESSION(fd fd_crypto, cmd const[CIOCGSESSION], arg ptr[in, session_op])
+ioctl$CIOCGSESSION2(fd fd_crypto, cmd const[CIOCGSESSION2], arg ptr[in, session2_op])
+ioctl$CIOCCRYPT(fd fd_crypto, cmd const[CIOCCRYPT], arg ptr[in, crypt_op])
+ioctl$CIOCCRYPTAEAD(fd fd_crypto, cmd const[CIOCCRYPTAEAD], arg ptr[in, crypt_aead])
+ioctl$CIOCFSESSION(fd fd_crypto, cmd const[CIOCFSESSION], arg ptr[in, array[int32]])
+
+crypt_find_op {
+	crid	int32
+	name	ptr[in, string]
+	namelen	const[32, int32]
```

Comment:
There is no `namelen` and `name` is not a pointer. I think this should just be `name string[32]`

Reply:
Okay. But can additional arguments be added? Or while running it will just give out an error?